lib/lightning_web/live/components/form.ex

```diff
@@ -409,12 +409,14 @@ defmodule LightningWeb.Components.Form do
   def select_field(assigns) do
     select_classes = ~w[
       mt-1
+      text-gray-700
       block
       w-full
-      rounded-md
+      rounded-lg
```

Comment:
Making this `text-gray-700` would make it different to the other inputs, for example the `text_input` is `text-gray-900` - if we're going to change it, it should be done for all of them that should match.
Why `rounded-lg` all of the other components use `rounded-md`?